### ISSUE TITLE
Resolve storybook warns

### ISSUE THREAD

**Issue by aaronzshey** [aaronzshey/react-hexgrid#2]
`npm run build-storybook` output:

```
> react-hexgrid@2.0.1 build-storybook
> build-storybook

info @storybook/react v6.5.9
info
info => Cleaning outputDir: /Users/aaron/react-hexgrid/storybook-static
(node:1197) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
info => Loading presets
info => Compiling manager..
info => Compiling preview..
info Addon-docs: using MDX1
info => Using implicit CSS loaders
info => Using default Webpack5 setup
17% building 2/14 entries 510/576 dependencies 12/210 modulesBrowserslist: caniuse-lite is outdated. Please run:
  npx update-browserslist-db@latest
  Why you should do it regularly: https://github.com/browserslist/update-db#readme
92% sealing asset processing TerserPlugininfo => Manager built (29 s)
info => Preview built (29 s)
WARN asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
WARN This can impact web performance.
WARN Assets:
WARN   863.13575e31.iframe.bundle.js (591 KiB)
WARN   566.1f80ae22.iframe.bundle.js (792 KiB)
WARN   840.339ca5fe.iframe.bundle.js (1.63 MiB)
WARN entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244 KiB). This can impact web performance.
WARN Entrypoints:
WARN   main (1.75 MiB)
WARN       runtime~main.52994ec7.iframe.bundle.js
WARN       840.339ca5fe.iframe.bundle.js
WARN       main.dc289003.iframe.bundle.js
WARN
```

**Comment by aaronzshey** [comment]
```
 Manual migration detected ───────────────────────────────────────────────────╮
│                                                                              │
│   Attention: We've detected that you're using argTypesRegex:                 │
│                                                                              │
│   .storybook/preview.js                                                      │
│   1 | export const parameters = {                                            │
│   > 2 |   actions: { argTypesRegex: "^on[A-Z].*" },                          │
│   |              ^^^^^^^^^^^^^                                               │
│   3 |   controls: {                                                          │
│   4 |     matchers: {                                                        │
│   5 |       color: /(background|color)$/i,                                   │
│                                                                              │
│   Storybook's play functions let you render your stories interactively.      │
│                                                                              │
│   In the past, play functions mocked action args implicitly by analyzing     │
│   the argTypesRegex                                                          │
│   in your preview.js|ts file.                                                │
│                                                                              │
│   However, Storybook 8 changes this behavior, and we now recommend using     │
│   the                                                                        │
│   (fn) function to mock your component's methods instead.                    │
│                                                                              │
│   Use the following command to check for implied mocked actions in your      │
│   play functions:                                                            │
│   npx storybook migrate find-implicit-spies                                  │
│   --glob="**/*.stories.@(js|jsx|ts|tsx)"                                     │
│                                                                              │
│   Then, refer to our docs to migrate your play functions to Storybook 8:     │
│   https://storybook.js.org/docs/8.0/essentials/actions#via-storybooktest-f   │
│   n-spy-function                                                             │
│                                                                              │
╰──────────────────────────────────────────────────────────────────────────────╯
```

**Comment by aaronzshey** [comment]
closed by #3
